### ISSUE TITLE
Multiple tooltips close to each other sometimes do not hide

### ISSUE THREAD

**Issue by evgenyboxer** [gilbarbara/react-tooltips#31]
>Expected behavior
The tooltip should hide when hovered quickly on multiple tooltips

>Actual behavior
The tooltips are not hidden... its a bit hard to reproduce, but basically if you do it quick enough it will not hide some of them.

>Steps to reproduce the problem
1. Create multiple tooltips.
2. Set the event type to hover on all of them.
3. Hover quickly on them

>React version
16.2

>React-Tooltips version
0.4.5

<img width="335" alt="screen shot 2018-03-17 at 11 28 17 pm" src="https://user-images.githubusercontent.com/254095/37555476-b9c63bea-2a3b-11e8-995e-59fb6fdd4ad5.png">

**Comment by gilbarbara** [comment]
Hey, are you using the default `eventDelay` prop?
Maybe setting it to 0 will be helpful in your case with lots of targets.

**Comment by evgenyboxer** [comment]
Yes that did solve the problem! However, it disappears quickly without any animation.
Do you think this is some how related to the fact mouse event?

**Comment by gilbarbara** [comment]
@evgenyboxer Yep, if the `event` is *hover* is a definitely a mouseleave issue.
